src/user-guide/writing-workflows/external-triggers.rst

```diff
@@ -142,26 +199,22 @@ It must be installed and run under the name *up*, as referenced in the
 .. literalinclude:: ../../workflows/xtrigger/workflow_state/downstream/flow.cylc
    :language: cylc
 
-Try starting the downstream workflow first, then the upstream, and
-watch what happens.
-In each cycle point the ``@upstream`` trigger in the downstream workflow
-waits on the task ``foo`` (with the same cycle point) in the upstream
-workflow to emit the *data ready* message.
-
-Some important points to note about this:
-
-- The function call interval, which determines how often the scheduler
-  checks the clock, is optional. Here it is
-  ``PT10S`` (i.e. 10 seconds, which is also the default value).
-- The ``workflow_state`` trigger function, like the
-  ``cylc workflow-state`` command, must have read-access to the upstream
-  workflow's public database.
-- The cycle point is supplied by a string template
-  ``%(point)s``. The string templates available to trigger functions
-  arguments are described in :ref:`Custom Trigger Functions`).
-
-The return value of the ``workflow_state`` trigger function looks like
-this:
+Try starting the downstream workflow first, then the upstream, and watch what
+happens. In each cycle point the ``@upstream`` trigger in the downstream workflow
+waits for the upstream task ``foo`` (with the same cycle point) workflow to generate
+the "data ready" message.
+
+Some important points to note:
+
+- The optional interval ``PT10S`` (10 seconds) determines how often the scheduler
```

Comment:
Is it worth having a warning here (or below this list) urgining users to consider what a sensible polling interval would be?

Reply:
I've moved this up to the general section too.